tasks/go.py

```diff
@@ -310,19 +331,6 @@ def lint_licenses(ctx, verbose=False):
 
     new_licenses = get_licenses_list(ctx)
 
-    if sys.platform == 'win32':
-        # ignore some licenses because we remove
-        # the deps in a hack for windows
-        ignore_licenses = ['github.com/shirou/gopsutil']
```

Comment:
Have you tried running `lint-licenses` on Windows to make sure this can be safely removed?

Reply:
So that change was okay but the `rm -rf` below obviously wasn't. I fixed that one!